docs/pages/includes/database-access/aws-bootstrap.mdx

```diff
@@ -0,0 +1,46 @@
+Teleport can bootstrap IAM permissions for the Database Service based on its
+configuration. You can use bootstrap in automatic or manual mode:
```

Comment:
"You can use bootstrap" suggests that "bootstrap" is the name for this feature. Is it? Should we capitalize it? Otherwise, to be clearer on the feature name, I think we can say something like,

```markdown
Teleport can bootstrap IAM permissions for the Database Service based on its
configuration using the `teleport db configure bootstrap` command. You can use
this command in automatic or manual mode:
```

Reply:
I think the more appropriate name would be "database configurator", but I like the idea of placing the command instead of the feature name.